src/BloomExe/MiscUI/BloomIntegrityDialog.cs

```diff
@@ -31,46 +26,46 @@ public BloomIntegrityDialog()
 		public static bool CheckIntegrity()
 		{
 			var errors = new StringBuilder();
-			var files = new[] { "Bloom.chm","PdfDroplet.exe", "Chorus.exe", "GeckofxHtmlToPdf.exe", "BloomBookUploader.exe", "optipng.exe" };
-			
-			var dirs = new[] {"AndikaNewBasic","factoryCollections","localization","xMatter","xslts"};
-			
+			var files = new[] { "Bloom.chm", "PdfDroplet.exe", "Chorus.exe", "GeckofxHtmlToPdf.exe", "BloomBookUploader.exe", "optipng.exe" };
+
+			var dirs = new[] { "AndikaNewBasic", "factoryCollections", "localization", "xMatter", "xslts" };
+
 			foreach(var fileName in files)
 			{
-				if (!Palaso.PlatformUtilities.Platform.IsWindows && fileName == "optipng.exe")
+				if(!Platform.IsWindows && fileName == "optipng.exe")
```

Comment:
Is it your editor or you (is there a difference?) that is removing the space here?  I find it uglier without the space, but wouldn't hold up a merge for that reason.